actors/builtin/miner/miner_actor.go

```diff
@@ -2067,6 +2068,291 @@ func (a Actor) RepayDebt(rt Runtime, _ *abi.EmptyValue) *abi.EmptyValue {
 	return nil
 }
 
+type ReplicaUpdate struct {
+	SectorID           abi.SectorNumber
+	Deadline           uint64
+	Partition          uint64
+	NewSealedSectorCID cid.Cid `checked:"true"`
```

Comment:
What exactly am I promising by slapping `checked` on here? That I sanity checked the input to be defined and the right prefix?

Reply:
[Pretty much see @magik6k's original PR and discussion](https://github.com/filecoin-project/specs-actors/pull/717)

Reply:
Gotcha, thanks!